paddle/fluid/pybind/imperative.cc

```diff
@@ -1929,9 +1929,6 @@ void BindImperative(py::module *m_ptr) {
              auto *t = self->MutableVar()->GetMutable<framework::LoDTensor>();
              auto *t_t =
                  target_t->MutableVar()->GetMutable<framework::LoDTensor>();
```

Comment:
It seems not a good interface name...

Reply:
Changed this interface name to _share_buffer_to.